### ISSUE TITLE
call without optional arguments fails

### ISSUE THREAD

**Issue by aremazeilles** [eurobench/eb_hum_bench#1]
I tried to launch the code without the optional argument, and the program fails: 

```term
docker run --rm -v $PWD/tests/reemc_conf:/conf -v $PWD/tests/input/2021_02_19/14/1:/in -v $PWD/output:/out pi_uhei:latest run_i3sa.py --conf /conf/robot.yaml --model /conf/reemc.lua --pos /in/pos.csv   --gait /in/gaitEvents.yaml --out /out

I3SA PI computation
Traceback (most recent call last):
  File "/usr/local/bin/run_i3sa.py", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/pi_runner/src/script/run_i3sa.py", line 38, in <module>
    experiment = Experiment(args)
  File "/home/pi_runner/src/locomotionbench/environment.py", line 399, in __init__
    self.files_not_provided.append(self.open_file('vel', args.vel, separator))
  File "/home/pi_runner/src/locomotionbench/environment.py", line 429, in open_file
    self.files[file_type] = pd.read_csv(file, sep=separator)
  File "/usr/local/lib/python3.6/dist-packages/pandas/io/parsers.py", line 688, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/local/lib/python3.6/dist-packages/pandas/io/parsers.py", line 437, in _read
    filepath_or_buffer, encoding, compression
  File "/usr/local/lib/python3.6/dist-packages/pandas/io/common.py", line 243, in get_filepath_or_buffer
    raise ValueError(msg)
ValueError: Invalid file path or buffer object type: <class 'NoneType'>
```

I made some other trials placing more optional parameters, but it seems the program expects all

**Comment by flxalr** [comment]
Fixed in [6783078](https://github.com/eurobench/eb_hum_bench/commit/67830782cc7afb9f2f4a69fa85abb366b00fcbfb)

**Comment by aremazeilles** [comment]
Correct, thanks!
